### PR TITLE
docs: add favicon

### PR DESCRIPTION
**Pull request by Kiarokh**
@adrianschmidt I was trying to add a favicon for lime elements, 
but don't know:
- how to link it properly, currently I don't see any favicon and tried different paths, but no success in seeing it in the browser 
- what to call the commit 

It'd be nice if you get a chance to look at this some day

## Review:
- [x] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [x] Commits have the correct *type* for the changes made
- [x] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

**Comment by adrianschmidt** [comment]
Yeah, I'll have a look after I've reviewed the dark-mode stuff 👍

**Comment by Kiarokh** [comment]
Thanks. No rush for this one ;) it can wait

File: src/img/favicon.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<svg viewBox="0 0 300 300" xmlns="http://www.w3.org/2000/svg">
+    <style>
+        #lime{fill:#4db6ac}#green{fill:#81c784;}#blue{fill:#4fc3f7;}#magenta{fill:#f34197;}#yellow{fill:#ffc107;}
+        @media (prefers-color-scheme:dark){
```
Comment (anderssonjohan) [inline src/img/favicon.svg:4]: @Kiarokh 😻

Comment (Kiarokh) [reply src/img/favicon.svg:4]: mihihihi… ☺️

**Review by adrianschmidt — APPROVED** [review]

**Pull request merged**